samples/outlook/50-recurrence/get-recurrence-appointment-organizer.yaml

```
@@ -0,0 +1,65 @@
+order: 3
+id: outlook-recurrence-get-recurrence-appointment-organizer
+name: Get recurrence (Appointment Organizer)
+description: Gets the recurrence pattern in Appointment Organizer mode.
+host: OUTLOOK
+api_set:
+    Mailbox: '1.7'
+script:
+    content: |
+        $("#get-recurrence").click(getRecurrence);
+
+        function getRecurrence() {
+          Office.context.mailbox.item.recurrence.getAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              var recurrence = asyncResult.value;
+              if (recurrence === null) {
+                console.log("This is a single appointment.");
+              } else {
+                console.log("Recurrence pattern: " + JSON.stringify(recurrence));
```

I found that just logging objects will result in ScriptLab UI that allows you to drill into JSON, which may be better than dumping JSON.stringify result?...

```suggestion
                console.log(recurrence);
```